src/server/routes/editor.js

```diff
@@ -512,4 +512,102 @@ router.post('/:id/achievements/', auth.isAuthenticated, (req, res) => {
 	handler.sendPromiseResult(res, rankPromise);
 });
 
+
+async function getOrderedCollections(from, size, entityType, req) {
+	const {UserCollection} = req.app.locals.orm;
+	const isThisCurrentUser = req.user && parseInt(req.params.id, 10) === parseInt(req.user.id, 10);
+
+	const allCollections = await new UserCollection()
+		.query((qb) => {
+			qb.leftJoin('bookbrainz.user_collection_collaborator',
+				'bookbrainz.user_collection.id', '=',
+				'bookbrainz.user_collection_collaborator.collection_id');
+		})
+		.where((builder) => {
+			if (!isThisCurrentUser) {
+				builder.where('public', true);
+			}
+			if (entityType) {
+				builder.where('entity_type', entityType);
+			}
+		})
+		.where((builder) => {
+			builder.where('editor_id', '=', req.params.id).orWhere('owner_id', '=', req.params.id);
```

Comment:
I am wondering if `user_collection_collaborator.editor_id` should be changed to `user_collection_collaborator.collaborator_id`